File: AcroStack/Services/Dtos/AuditLogging/AuditLogActionDto.cs
using System;

namespace AcroStack.Services.Dtos.AuditLogging;

public class AuditLogActionDto
{
    public Guid Id { get; set; }
    public Guid AuditLogId { get; set; }
    public string? ServiceName { get; set; }
    public string? MethodName { get; set; }
    public string? Parameters { get; set; }
    public DateTime ExecutionTime { get; set; }
    public int ExecutionDuration { get; set; }
    public Guid? TenantId { get; set; }
}

File: react/src/api/models/acroStack/services/dtos/auditLogging/AuditLogActionDto.ts
/* oxlint-disable */

/**
 * @type object
 */
export type AcroStackServicesDtosAuditLoggingAuditLogActionDto = {
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  id?: string;
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  auditLogId?: string;
  /**
   * @type string | undefined
   */
  serviceName?: string | null;
  /**
   * @type string | undefined
   */
  methodName?: string | null;
  /**
   * @type string | undefined
   */
  parameters?: string | null;
  /**
   * @description
   * Format: `date-time`
   * @type string | undefined
   */
  executionTime?: string;
  /**
   * @description
   * Format: `int32`
   * @type integer | undefined
   */
  executionDuration?: number;
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  tenantId?: string | null;
};
